src/main/scala/net/cakesolutions/CakeDockerPlugin.scala

```diff
@@ -24,6 +19,19 @@ object CakeDockerPlugin extends AutoPlugin {
     dockerRepository := None,
     packageName in Docker := name.value,
     maintainer in Docker := "Cake Solutions <devops@cakesolutions.net>",
-    version in Docker := sys.props.get("tag").getOrElse(version.value)
+    version in Docker := sys.props.get("tag").getOrElse(version.value),
+    dockerCommands ++= imageLabels(name.value)
   )
+
+  private def imageLabels(containerName: String): Seq[Cmd] = {
+    val dockerArgList =
+      CakeBuildInfoKeys.generalInfo ++ CakeBuildInfoKeys.dockerInfo
+
+    dockerArgList
```

Comment:
In order to reduce the Docker image size, we should simply generate a single `LABEL` here